build/scripts/entrypoint-volume.sh

```diff
@@ -66,6 +66,44 @@ if [ -n "${OPENVSX_REGISTRY_URL+x}" ]; then
   sed -i -e "s|serviceUrl:\".*\",itemUrl:\".*\"},version|serviceUrl:\"${OPENVSX_URL}/gallery\",itemUrl:\"${OPENVSX_URL}/item\"},version|" out/vs/workbench/workbench.web.main.js
 fi
 
+
+# To switch the webview content being loaded from local workspace we need to check for 'WEBVIEW_LOCAL_RESOURCES' environment variable.
+# If the variable is set, then:
+if [ -n "${WEBVIEW_LOCAL_RESOURCES+x}" ]; then
+  # fetch che-code endpoint value from /devworkspace-metadata/flattened.devworkspace.yaml
+  CHE_CODE_ENDPOINT=$(yq -r '
+    .components
+    | to_entries[]
+    | select(.value.container.command[0] == "/checode/entrypoint-volume.sh")
+    | .value.container.endpoints
+    | to_entries[]
+    | select(.value.name == "che-code")
+    | .value.attributes
+    | to_entries[]
+    | select(.key == "controller.devfile.io/endpoint-url")
+    | .value
+  ' /devworkspace-metadata/flattened.devworkspace.yaml)
+
+  # make a valid base URI for webview static resources
+  WEBVIEW_CONTENT_BASE_URL=$CHE_CODE_ENDPOINT"oss-dev/static/out/vs/workbench/contrib/webview/browser/pre/"
+  echo "WEBVIEW_CONTENT_BASE_URL: $WEBVIEW_CONTENT_BASE_URL"
+
+  # fetch the existing base URI from product.json
+  WEBVIEW_CONTENT_BASE_URL_TEMPLATE=$(jq -r '.webviewContentExternalBaseUrlTemplate' /checode/checode-linux-libc/product.json)
+  echo "WEBVIEW_CONTENT_BASE_URL_TEMPLATE: $WEBVIEW_CONTENT_BASE_URL_TEMPLATE"
+
+  # escape special characters
+  WORK_TEMPLATE=$WEBVIEW_CONTENT_BASE_URL_TEMPLATE
+  WORK_TEMPLATE=${WORK_TEMPLATE//[\{]/\\{}
+  WORK_TEMPLATE=${WORK_TEMPLATE//[\}]/\\\}}
+
+  # apply new base URI to the following files
+  sed -i -r -e "s|${WORK_TEMPLATE}|${WEBVIEW_CONTENT_BASE_URL}|" out/vs/workbench/workbench.web.main.js
+  sed -i -r -e "s|${WORK_TEMPLATE}|${WEBVIEW_CONTENT_BASE_URL}|" out/vs/workbench/api/node/extensionHostProcess.js
+  sed -i -r -e "s|${WORK_TEMPLATE}|${WEBVIEW_CONTENT_BASE_URL}|" product.json
```

Comment:
To make it clear, I'm suggesting adding a short comment here that the actual value of `webviewContentExternalBaseUrlTemplate` in the `product.json` has no effect at runtime, and it serves as a marker for easily verifying the source of the webview static resources.

Reply:
Done